### PR TITLE
XSIAM main playbooks

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## fixes
fixes: https://github.com/demisto/etc/issues/46337
fixes: https://github.com/demisto/etc/issues/46332
fixes: https://github.com/demisto/etc/issues/46331

## Description
The new main playbooks for XSIAM

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-NGFW_Scan_README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* Entity enrichment
+
+**Response Actions**
+
+The playbook's response actions are based on the initial data provided within the alert. In that phase, the playbook will execute:
+
+* Auto block IP address
+* Report IP address (If configured as true in the playbook inputs)
+
+When the playbook proceeds, it checks for additional activity using the Endpoint Investigation Plan playbook, and another phase, which includes the Containment Plan playbook, is executed.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-NGFW_Scan_README.md:23]: When the playbook proceeds, it checks for additional activity using the Endpoint Investigation Plan playbook, based on the data or the information found, the Containment Plan playbook will execute again.
In this phase, the playbook will execute the following containment actions:


I think that is clearer, wdyt?

File: Packs/Core/Playbooks/playbook-NGFW_Scan_README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+| SOCEmailAddress | The SOC email address. |  | Optional |
+| reportIPAddress | Whether to report the IP address to AbuseIPDB or not. | false | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![NGFW Scan](Insert the link to your image here)
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-NGFW_Scan_README.md]: fixed

File: Packs/Core/Playbooks/playbook-NGFW_Scan_README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* NGFW Internal Scan playbook
+* Endpoint Investigation Plan playbook
+* Entity enrichment
+
+**Response Actions**
+
+The playbook's response actions are based on the initial data provided within the alert. In that phase, the playbook will execute:
+
+* Auto block IP address
+* Report IP address (If configured as true in the playbook inputs)
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-NGFW_Scan_README.md:21]: report to AbuseIPDB

File: Packs/Core/Playbooks/playbook-NGFW_Scan_README.md (hunk 1 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-NGFW_Scan_README.md:23]: nope, I had something written like yours but Grammarly fixed it. now, I think the current is better.

File: Packs/Core/Playbooks/playbook-T1036_-_Masquerading_README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+This playbook handles masquerading alerts based on MITRE T1036 technique.
+An attacker might leverage Microsoft Windows well-known image names to run malicious processes without being caught.
+
+**Attacker's Goals:**
+
+An attacker is attempting to masquerade as standard windows images by using a trusted name to execute malicious code.
+
+**Investigative Actions:**
+
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-T1036_-_Masquerading_README.md:9]: analysis can be part of the whole investigative actions

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**